src/norm/private/postgres/rowutils.nim

```diff
@@ -14,40 +14,54 @@ when (NimMajor, NimMinor) <= (1, 6):
 else:
   import std/macros
 
-proc fromRowPos[T: Model](obj: var T, row: Row, pos: var Natural, skip = false) =
+
+func isContainer*[T](val: typedesc[T]): bool {.compileTime.} = T is ref object
+func isContainer*[T](val: T): bool {.compileTime.} = T is ref object
+func isContainer*[T](val: typedesc[Option[T]]): bool {.compileTime.} = T is ref object  
+func isContainer*[T](val: Option[T]): bool {.compileTime.} = T is ref object  
+
+func toOptional*[T: ref object](val: T): Option[T] = some val
+func toOptional*[T: ref object](val: Option[T]): Option[T] = val
+func isEmptyColumn(row: Row, index: int): bool = row[index].kind == dvkNull
+
```

Comment:
As stated, the procs above have 2 problems:
1) I don't think they belong in rowutils, I'm just not sure where else to put them. Ideally something that is shared between postgres and sqlite packages since they are **identical** between the packages
2) I can't think of better names for `isContainer` and `toOptional`. If anyone has any, please suggest.